src/components/15Years/15YearsTimeline.vue

```diff
@@ -241,18 +241,15 @@ export default {
 
 <style lang="scss" scoped>
 @use 'sass:math';
-@import '#src/assets/scss/settings';
-@import '#src/assets/scss/components/15-years/15-years';
+@use '#src/assets/scss/settings' as *;
+@use '#src/assets/scss/components/15-years/15-years' as *;
 
 @mixin timeline-link() {
 	@include link();
 
-	color: $mint;
-
-	// &:hover {
-	// 	background-color: $offwhite;
-	// 	color: $twilight;
-	// }
+	& {
+		color: $mint;
+	}
```

Comment:
This is the one case where the rule still needed to come after the include, so it uses the suggested `& { ... }` syntax.